frontend/src/general/LogoutMenu/LogoutMenu.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  ClickAwayListener,
+  Grow,
+  MenuItem,
+  MenuList,
+  Paper,
+  Popper,
+  Typography
+} from "@material-ui/core";
+import { NavLink } from "react-router-dom";
+
+export default function LogoutMenu(props) {
+  const { anchor, open, handleClose, handleLogout } = props;
+
+  return (
+    <Popper open={open} anchorEl={anchor} transition disablePortal>
+      <Grow in={open} style={{ transformOrigin: "center top" }}>
+        <Paper>
+          <ClickAwayListener onClickAway={handleClose}>
+            <MenuList autoFocusItem={open} id="menu-list-grow">
+              <MenuItem disabled>
+                <Typography color="textSecondary">Quick Actions</Typography>
```

Comment:
point of this?

Reply:
It seems like it's supposed to act as a title for the dropdown, but imo it feels more like a disabled option since it's paler text.

Reply:
it's a title since it's weird to have a drop down that has dahsboard, my devices and logout